files/README.md

```diff
@@ -0,0 +1,98 @@
+## Downloaded packages
+
+### SHA-256
+
+For each uploaded _Debian_ package a [SHA-256](https://en.wikipedia.org/wiki/SHA-2) checksum is computed.
+This checksum is used to create a file tree where each package will reside, with following hierarchy.
+
+#### Directory and sub-directories structure
+
+  - 1st and 2nd characters of SHA-256 checksum as sub-directory of **rootDir**/pool directory.
+  - 3rd and 4th characters of SHA-256 checksum as sub-directory of the former
+
+ex:
+
+ sha256sum **476e0**cdac6bc757dd2b78bacc1325323b09c45ecb41d4562deec2a1c7c148405 my-package_1.2.3_all.deb
+
+```bash
+${rootDir}/pool # rootDir defined in aptly.conf
+└── 47
+    └── 6e
+```
+
+#### Filename
+
+The following items are concatenated to form the filename under which package is stored.
+  - 5th to the 31st characters of SHA-256 checksum
+  - "\_" (undescore) 
+  - filename of uploaded _Debian_ as defined in [Debian package file names](https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_debian_package_file_names)
+
+ex:
+
+ sha256sum 476e0**cdac6bc757dd2b78bacc13253**23b09c45ecb41d4562deec2a1c7c148405 **my-package_1.2.3_all.deb**
+
+```
+ 0cdac6bc757dd2b78bacc13253_my-package_1.2.3_all.deb
+```
+
+### MD5
```

Comment:
this is actually legacy layout, problem was that it leads to real-world collisions when two first octets and filename are identical. 

aptly still supports it, but only for 'reading' from the pool, it never puts files this way for new package files

Reply:
There were a note about MD5 being _legacy_ in the [README.md](https://github.com/aptly-dev/aptly/blob/1c912e7c0b2d8d80275afc637059dd9fac1301ba/files/README.md) that I have updated.
Let me know if it's enough

Reply:
thanks, looks good!